hwy/abort_test.cc

```diff
@@ -0,0 +1,63 @@
+// Copyright 2024 Arm Limited and/or its affiliates <open-source-office@arm.com>
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include "hwy/abort.h"
+
+#include "gtest/gtest.h"
```

Comment:
Please change to "hwy/tests/hwy_gtest.h" - our internal MSVC is no longer able to build with GTest.